src/main/java/org/prebid/server/spring/config/bidder/EmxDigitalConfiguration.java

```diff
@@ -19,7 +19,7 @@
 @PropertySource(value = "classpath:/bidder-config/emxdigital.yaml", factory = YamlPropertySourceFactory.class)
 public class EmxDigitalConfiguration {
 
-    private static final String BIDDER_NAME = "emx_digital";
+    private static final String BIDDER_NAME = "emxdigital";
```

Comment:
Why this was changed?

Reply:
I spot some inconsistencies and failures related to the bidder name, yaml and json files names, so I decided to rename the bidder for good